tests/testTicket2352.py

```diff
@@ -51,15 +54,18 @@ def testExtName(self):
         self.checkExtName("THREE", 3, 4)
 
     def checkExtNum(self, hdu, extNum):
+        if hdu is None:
+            hdu = -(1 << 31)            # == INT_MIN
```

Comment:
Can we get the `INT_MIN` -> `None` translation at the pybind11 level, so we don't have to do this?